src/site/layouts/health_care_region_page.drupal.liquid

```diff
@@ -138,73 +140,108 @@ Example data:
       <article class="usa-content">
         <h1>{{ title }}</h1>
         {% assign image = fieldMedia.entity.image %}
-        <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="{{ image.derivative.width }}" height="{{ image.derivative.height }}">
-        <div class="usa-grid usa-grid-full">
-          <div class="usa-width-one-third">
-            Make an appointment
+        <img src="{{ image.derivative.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="100%">
+        <div class="usa-grid usa-grid-full vads-u-margin-top--1p5">
+          <div class="usa-width-one-third facility-main-cta-item">
+            <a href="{{ entityUrl.path }}/appointments">Make an appointment</a>
```

Comment:
Is the design such that just the text inside these blue boxes is clickable, or should the whole box be a link? It feels a little better to me that way, but I'm not sure if the design says otherwise (I think this is a new pattern?).